observability/logging/cluster-logging-support.adoc

```diff
@@ -34,19 +34,31 @@ LokiStack development is ongoing. Not all APIs are currently supported.
 
 |LokiStack
 |lokistack.loki.grafana.com/v1
-|Supported in 5.5
+|Supported from 5.5
 
 |RulerConfig
 |rulerconfig.loki.grafana/v1
-|Supported in 5.7
+|Supported from 5.7
 
 |AlertingRule
 |alertingrule.loki.grafana/v1
-|Supported in 5.7
+|Supported from 5.7
 
 |RecordingRule
 |recordingrule.loki.grafana/v1
-|Supported in 5.7
+|Supported from 5.7
+
+|LogFileMetricExporter
+|LogFileMetricExporter.logging.openshift.io/v1alpha1
+|Supported from 5.8
+
+|ClusterLogForwarder
+|clusterlogforwarder.observability.openshift.io/v1
+|Supported from 5.8
```

Comment:
This I believe this is also not correct as this API is older than 5.8, @jcantrill might know the exact number